docs/docs/domain.mdx

```diff
@@ -260,44 +288,35 @@ Make this a nice table instead.
       type: list
   ```
 
-
-
 * **Description**
 
-  The feature of this slot is set to `1` if a value with a list is set,
-  where the list is not empty. If no value is set, or the empty list is the
-  set value, the feature will be `0`. The **length of the list stored in
+  If `influence_conversation` is set to `true`, the assistant's behavior will change
+  depending on whether the list is empty or not. The **length of the list stored in
   the slot does not influence the dialogue**.
 
```

Comment:
adding this note because it's  not mentioned anywhere
```suggestion

If a `list` slot is [auto filled by an entity of the same name](domain.mdx#slot-auto-fill), each entity value will be appended to the value of the slot. 

```

Reply:
Or it could go under "auto fill slots"

Reply:
@melindaloubser1 This means if there are previous values stored, they won't be overridden?

Reply:
I believe so? That's what I thought, but one way or another we should document the behaviour

Reply:
Ok nope, that's what [this forum post]() implied, but I checked and it overwrites.
So this type is only really useful for custom actions right?

Reply:
your link doesn't link to the forum post 😆

Reply:
Judging from the code it should be replaced and not extended

Reply:
Oh ugh now I can't find it, but yes it seems like that's always been the case